conftest.py

```diff
@@ -243,7 +226,7 @@ def hashing(request):
         _save_result(hashes, request.cls.answer_file)
     # Compare hashes
     elif not no_hash and not store_hash:
-        _compare_result(hashes, request.cls.saved_hashes)
+        assert hashes == request.cls.saved_hashes
```

Comment:
come to think of it I'm not sure it's good practice to have assert statements in a fixture. Is there any resource available on the topic ?

Reply:
Not sure. I'll have a look.

Reply:
So I didn't manage to find anything either way. However, none of the examples I saw in the pytest documentation do it, so I'm guessing it's just implicitly frowned upon. However, I don't see why we can't have the `hashing` fixture call `_compare_result` and, if an `AssertionError` is raised then have `hashing` call `pytest.fail()` in order to mark the test as failing, not let the `AssertionError` pass silently, but also not stop the pytest run. Thoughts?

Reply:
So I tried running this
```python
import pytest

@pytest.fixture()
def orwell():
    assert 2 + 2 == 5

def test_dystopia(orwell):
    assert 1 + 1 == 2
```

and learned that this doesn't "FAIL", but `ERROR`s out. I don't think it's reasonable to alter pytest's indications; if the authors wanted to make a distinction between failure and error, let's not mingle with it.
Considering this, I'd advise against raising `pytest.fail`, but I can actually get along with the (current ?) implementation I first commented. I see that it's now outdated so I'm not even sure this is relevant anymore, sorry :/

Reply:
@jcoughlin11 ... is it ?

Reply:
@jcoughlin11 is this still relevant to this PR?

Reply:
@neutrinoceros Sorry, I missed your response! I looked, and it is still relevant, since I currently do an `assert` in the fixture and, if there's an error, raises `pytest.fail()`. I also just saw that this was merged, so I can change it in the next pytest PR, if needed.